test/controllers/token_controller_test.exs

```diff
@@ -77,10 +73,11 @@ defmodule CodeCorps.TokenControllerTest do
       refute response["token"]
       refute response["user_id"]
       [error | _] = response["errors"]
-      assert error["id"] == "UNAUTHORIZED"
-      assert error["title"] == "401 Unauthorized"
-      assert error["status"] == 401
+      assert is_unauthorized?(error)
       assert error["detail"] == "token_expired"
     end
   end
+
+  defp is_unauthorized?(%{"id" => "UNAUTHORIZED", "title" => "401 Unauthorized", "status" => 401}), do: true
```

Comment:
Opted to leave these methods here, as I didn't see any other tests making this assertion. Happy to extract if there is another use-case, but as it seems specific to this test, will leave here for now.